content/en/contribute/code/core/dev-environment.md

```diff
@@ -136,54 +135,29 @@ services:
           - "5984:5984"
           - "5986:5986"
 EOF
-```
+{{< /tab >}}
+{{< tab header="macOS" >}}
+docker build -t couchdb-apple-silicon ~/cht-core/couchdb/.
```

Comment:
This command only works if `cht-core` is cloned in the `~/` directory. Can we specify that? Or advise the readers to `cd` into `cht-core` and run `docker build -t couchdb-apple-silicon couchdb/.` instead?

Reply:
Ah - good thinking! The whole guide is predicated on using the default location of `~/cht-core/` - I'll add a heads up that this continues to use that.  

The guide is meant to be followed all in one go with a "happy path" of `~/cht-core/`, but we can call this out for folks who didn't notice.

Reply:
Perfect, then just removing the `~/cht-core/` will help, as the readers will already be in the `cht-core` directory. I will add a suggestion.